### PR TITLE
Update events-and-binds.md

### PR DESCRIPTION
**Pull request by Zoh63392187**
Invalid function -> end statement missing

### CODE REVIEW

**Comment by brainiac** [comment]
This isn’t correct, only the first line is conditional

**Comment by Zoh63392187** [comment]
I guess my wording is not correct (Stupid Dane)

File: lua/events-and-binds.md [hunk 1]
```diff
@@ -56,8 +56,10 @@ local callback = function (...)
     local args = {...}
     local str = ''
     for i = 1, #args, 1 do
-        if i > 1 then str = str .. ' '
-        str = str .. args[i]
+        if i > 1 then
+            str = str .. ' '
+            str = str .. args[i]
```
Comment (brainiac) [inline lua/events-and-binds.md]: this shouldn't be part of the condition. the condition was to add a space between arguments after the first.

**Review by brainiac — CHANGES_REQUESTED** [review]

**Review by Zoh63392187 — COMMENTED** [review]
All good

**Pull request merged**